### PR TITLE
Adapt Codenvy to changes in Che

### PR DESCRIPTION
**Pull request by garagatyi**
### What does this PR do?
Adapt Codenvy to changes introduced by eclipse/che#2108

### Tests written?
Yes

### CODE REVIEW

**Comment by garagatyi** [comment]
@evoevodin @skabashnyuk @sleshchenko @akorneta @mmorhun Please review

**Comment by skabashnyuk** [comment]
ok

File: wsmaster/codenvy-hosted-api-workspace/src/main/java/com/codenvy/api/workspace/server/filters/WorkspacePermissionsFilter.java [hunk 1]
```diff
@@ -106,13 +114,26 @@ public void filter(GenericResourceMethod genericResourceMethod, Object[] argumen
...
                 return;
+            }
+
+            case "getMachines":
+            case "executeCommandInMachine":
+            case "getProcesses":
+            case "stopProcess":
+            case "getProcessLogs":
+            case "getMachineById": {
+                workspaceId = ((String)arguments[0]);
```
Comment (sleshchenko) [inline wsmaster/codenvy-hosted-api-workspace/src/main/java/com/codenvy/api/workspace/server/filters/WorkspacePermissionsFilter.java]: cover this usecases with tests

Comment (sleshchenko) [inline wsmaster/codenvy-hosted-api-workspace/src/main/java/com/codenvy/api/workspace/server/filters/WorkspacePermissionsFilter.java]: I think you can move and adapt these tests https://github.com/codenvy/codenvy/pull/571/files#diff-9d3d8030c3b0a013de5e4c9aa51cc7f2L116

**Comment by sleshchenko** [comment]
other OK for me

**Comment by mmorhun** [comment]
+1

**Comment by akorneta** [comment]
ok

**Comment by garagatyi** [comment]
Updated PR with all discussed fixes and improvements

**Comment by evoevodin** [comment]
ok

**Pull request merged**